src/utils/utils_posix_common.c

```diff
@@ -37,13 +37,17 @@
          ? LLONG_MAX                                                           \
          : (sizeof(off_t) == sizeof(long) ? LONG_MAX : INT_MAX))
 
-static UTIL_ONCE_FLAG Page_size_is_initialized = UTIL_ONCE_FLAG_INIT;
+static UTIL_ONCE_FLAG System_info_is_initialized = UTIL_ONCE_FLAG_INIT;
 static size_t Page_size;
+static unsigned Core_conut;
```

Comment:
You could pack these two values into a neat system_info struct

Reply:
I'm not convinced if it is worth extra structure, if there will be more fields then sure, but for two?...